website/blog/2024-03-11-AutoDefense/Defending LLMs Against Jailbreak Attacks with AutoDefense.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: AutoDefense - Defend against jailbreak attacks with AutoGen
+authors:
+- yifanzeng
+- yiranwu
+tags: [LLM, GPT, research]
+---
+
+![architecture](imgs/architecture.png)
+
+## TL;DR
+
+- We propose **AutoDefense**, a multi-agent defense framework using AutoGen to protect LLMs from jailbreak attacks.
+- AutoDefense employs a response-filtering mechanism with specialized LLM agents collaborating to analyze potentially harmful responses.
+- Experiments show our three-agents (consisting of an intention analyzer, a prompt analyzer, and a judge) defense agency with LLaMA-2-13B effectively reduces jailbreak attack success rate while maintaining low false positives on normal user requests.
+
```

Comment:
Add a section "## What is a jailbreak attack?" to introduce jailbreak attacks use easier examples. 

```
LLMs are pre-trained with moral alignment to be prohibited from generating harmful information at user request. 

(Example 1: ask a harmful question but the LLM rejected. Retrieve the question and answer)

However, fine-crafted jailbreak prompts can trick LLMs into answering these questions: 

(Example 2: the harmful question with partial of the harmful question)

Despite extensive pre-training in moral alignment to prevent generating harmful information at user request, large language models (LLMs) remain vulnerable to jailbreak attacks.
```